server/sonar-db-dao/src/test/java/org/sonar/db/purge/PurgeDaoTest.java

```diff
@@ -222,13 +239,19 @@ public void deleteProject_deletes_webhook_deliveries() {
     assertThat(selectAllDeliveryUuids(dbTester, dbSession)).containsOnly("D2");
   }
 
-  private CeActivityDto insertCeActivity(String componentUuid) {
+  private CeQueueDto createCeQueue(ComponentDto component) {
     CeQueueDto queueDto = new CeQueueDto();
     queueDto.setUuid(Uuids.create());
     queueDto.setTaskType(REPORT);
-    queueDto.setComponentUuid(componentUuid);
+    queueDto.setComponentUuid(component.uuid());
     queueDto.setSubmitterLogin("henri");
     queueDto.setCreatedAt(1_300_000_000_000L);
+    queueDto.setStatus(CeQueueDto.Status.PENDING);
```

Comment:
you should randomly pick a `CeQueueDto.Status` value, that would enforce that the value doesn't matter